sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -278,6 +278,20 @@ private static class ClientPrivateKeyStrategy implements PrivateKeyStrategy {
 }
 ```
 
+
+### Enable local JRE key store
+
+If you use key vault key store but don't override the trust manager factory, by default, the JRE trust manager will be used and will employ only the key vault key store. In this case the local JRE key store will not work. To let it also work with Key Vault key store, you need use the key vault trust manager which employs both the key vault key store and the local JRE keystore. To do this, add the following configuration:
```

Comment:
We need to discuss about this.

Reply:
Can we put all certs in JRE key store into KeyvaultKeyStore? So we can support the certs in local JRE key store without override trust manager factory.

Reply:
> Can we put all certs in JRE key store into KeyvaultKeyStore? So we can support the certs in local JRE key store without override trust manager factory.

no need in this way, if you want to implement this in  key store level, just let key vault key store hold the jre key store as an assistance delegate.